src/status_im/multiaccounts/login/core.cljs

```diff
@@ -190,6 +191,11 @@
         (data-store.settings/rpc->settings settings)
         multiaccount (dissoc settings :networks/current-network :networks/networks)
         network-id (str (get-in networks [current-network :config :NetworkId]))]
+    (log/set-level! (if-let [log-level (get multiaccount :log-level)]
+                      (-> (if (string/blank? log-level) "ERROR" log-level)
+                          string/lower-case
+                          keyword)
+                      config/log-level))
```

Comment:
Alright, couple things to note here:

- When logging in, there's a possibility that the account in question doesn't have an actual `log-level` yet. In that case, we're going with `config/log-level` which is also what's being set during app bootstrap
- If there is a log-level, there's a possibility that it's an empty string. This is the case when logging was set to "DISABLED". I couldn't find a way to actually disable the logger so I'm defaulting to "ERROR" here

Reply:
Since this is a side-effect, you should create a set-level! effect in logging.core with reg-event-fx and use that effect here.
Also as a default you should probably use `config/log-level-status-go` and assoc the log-level in the db here so that you don't need to change the subscription. You simplify the subscription by simply returning `(get multiaccount :log-level)` since it will then always have a log-level.
Would also be nice to put a default log-level for newly created accounts since it seems it is not the case otherwise you would always have a log-level here.